src/Juvix/Compiler/Pipeline/EntryPoint.hs

```diff
@@ -44,8 +45,9 @@ defaultEntryPoint root mainFile =
       _entryPointStdin = Nothing,
       _entryPointPackage = defaultPackage root buildDir,
       _entryPointGenericOptions = defaultGenericOptions,
-      _entryPointTarget = TargetCNative64,
```

Comment:
was this change intended?

Reply:
It shouldn't matter, because if you need a different target you should override it. I tried to make defaults consistent.